src/Resources/assets/shop/js/initAutocomleate.js

```diff
@@ -0,0 +1,3 @@
+import ElasticSearchAutocomplete from './elasticSearchAutocomplete';
+
+new ElasticSearchAutocomplete().init();
```

Comment:
missing new line, missing if statement - what if there is no page with required search button? it will throw errors, but its not needed .